docs/pages/x.tsx

```diff
@@ -27,6 +28,7 @@ export default function Home() {
         <XComponents />
         <XDataGrid />
         <XTheming />
+        <XPlans />
```

Comment:
I love how it looks

<img width="1255" alt="Screenshot 2021-11-29 at 16 28 43" src="https://user-images.githubusercontent.com/3165635/143895840-c8fccd35-d2a6-4b15-9a43-bedb3cb12aee.png">